src/angularjs/src/app/places/list/place-list.html

```diff
@@ -69,8 +81,7 @@ <h2 class="sidebar-title">City/Town Search</h2>
         <section>
```

Comment:
Looks like the section headers aren't implemented yet, I made a separate issue for that: #326